### ISSUE TITLE
Navbar tab improvements

### ISSUE THREAD

**Issue by martonlanga** [termyapp/Termy#78]
- [ ] Stuff in Notion
 - [ ] Windows pointer cursor
 - [ ] New Tab

**Comment by ThatOneCalculator** [comment]
Not a fan of the non-system navbar. Interferes with my WM, and acts as more of an annoyance than anything else.

OS: Arch
WM: bspwm

**Comment by martonlanga** [comment]
Could you please send a screenshot of what a system navbar looks like?

Would having an option to hide it solve this problem for you?

**Comment by ThatOneCalculator** [comment]
Definitely. I'm on bspwm and have chosen to hide title bars, but I tried it out on GNOME and it didn't respect it there either
